Tests/Tests/Core/Utilities/UserDefaults/UserDefaultTests.swift

```diff
@@ -36,17 +35,37 @@ class UserDefaultTests: XCTestCase {
     // Then
     XCTAssertEqual(userDefaults.bool(forKey: Defaults.testBoolKey), Defaults.isAuthenticated)
   }
+  
+  func testSaveDataValue() {
+    let givenValue: Data = .init()
+    
+    Defaults.profileData = givenValue
+    
+    XCTAssertNotNil(Defaults.profileData)
+  }
+  
+  func testSaveDataNullValue() {
+    let givenValue: Data? = nil
+    
+    Defaults.profileData = givenValue
```

Comment:
This crashes. @EgzonArifi can you please take a look what's going on?

Reply:
Should be resolved.